go.mod

```diff
@@ -3,15 +3,15 @@ module sigs.k8s.io/security-profiles-operator
 go 1.15
 
 require (
-	github.com/containers/common v0.21.0 // can be built with the `seccomp` build tag to support more features
```

Comment:
Removed that comment because it's unlikely that we will ever depend on this build tag.